components/typography.css

```diff
@@ -0,0 +1,151 @@
+/* heading */
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+.h6,
+.h1,
+.h2,
+.h3,
+.h4,
+.h5 {
+  line-height: 1.5;
+  margin: 0.625rem 0;
+  font-weight: bold;
+}
+
+p {
+  line-height: 1.5;
+  margin: 0.625rem 0;
+}
+
+h1,
+.h1 {
+  font-size: 2rem;
+}
+
+h2,
+.h2 {
+  font-size: 1.75rem;
+}
+
+h3,
+.h3 {
+  font-size: 1.5rem;
+}
+
+h4,
+.h4 {
+  font-size: 1.25rem;
+}
+
+h5,
+.h5 {
+  font-size: 1rem;
+}
+
+h6,
+.h6 {
+  font-size: 1.875rem;
+}
+
+/* txt size*/
+
+p,
+.txt-rg {
+  font-size: 1rem;
+  margin: 1rem 0;
+}
+
+.txt-lg {
+  font-size: 1.875rem;
+}
+
+.txt-md {
+  font-size: 1.5rem;
+}
+
+.txt-sm {
+  font-size: 0.875rem;
+}
+
+/* font weights */
+
+.txt-light {
+  font-weight: 300;
+}
+
+.txt-regular {
+  font-weight: 400;
+}
+
+.txt-semibold {
+  font-weight: 600;
+}
+
+.txt-bold {
+  font-weight: 700;
+}
+
+.txt-black {
+  font-weight: 900;
+}
+
+/* txt utility */
+
+.secondary-txt {
```

Comment:
use consistent naming convention, in this case, it should be txt-secondary since you have used this naming convention .txt-something for rest of the classes